### PR TITLE
A merge guide for updating webrtc-pc

### PR DESCRIPTION
**Pull request by alvestrand**
Proposal. Sketchy.

### CODE REVIEW

File: merge-guide.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+In order to have a new feature considered for inclusion, write it up as an extension spec
+or as a pull request against webrtc-extensions.
+
+## Landing in webrtc-extensions
+
+We will land extensions in webrtc-extensions on the following criteria:
+
+* There is rough consensus in the WG that this API makes sense to consider
+* There is commitment from at least one browser vendor that they intend to implement
```
Comment (dontcallmedom) [inline merge-guide.md:24]: I think this should also say what goes into webrtc-extensions vs an extension spec - my sense is anything that can be defined cleanly as an extension spec should be defined that way (e.g. doesn't require monkey patching algorithms).

**Review by dontcallmedom — COMMENTED** [review]
this lgtm, with possibly some clarification needed on webrtc-extensions vs extension specs

File: merge-guide.md (hunk 1 above)
Comment (alvestrand) [reply merge-guide.md:24]: I am thinking that "large features" (webrtc-encoded-transform) should be their own spec. "Small features" (priority would have been one if -extensions existed at the time, header control is one) should go into the extensions spec.

Monkey-patching will be equally hard in extensions and in standalone specs.

**Comment by alvestrand** [comment]
@dontcallmedom is there a preferred format / template for this sort of document?

**Comment by dontcallmedom** [comment]
I realize there is an aspect this document doesn't cover: where issues should be filed between -pc and -extensions.

In terms of format, I don't think there is any particular template to follow; maybe we could put this into the README.md instead of a separate file (and if not, the README should prominently link to it).

**Comment by alvestrand** [comment]
I added text to the README.md pointing here, and text to this document talking about where to file issues. Does it look ready now?

**Review by dontcallmedom — APPROVED** [review]
LGTM

**Pull request merged**